doc/whatsnew/v1.3.0.rst

```diff
@@ -50,3 +63,9 @@ Fixes
 * An invalid encoding is now replaced by the default encoding, if
   ``config.enforce_valid_values`` is not set (:issue:`815`)
 * Correctly handle elements with ambiguous VR in sequence items (:issue:`804`)
+* Fix bug where new DicomDir objects always have is_implicit_VR
+* Fix dataset equality for mixed raw vs converted data elements (:issue:`835`)
+* Remove excess padding in Pixel Data 
+* Fix wrong date format in anonymize example
+* Fix unknown VR exception message when VR isn't ASCII (:issue:`791`)
+
```

Comment:
I think we forgot to add the fix for the Japonese encoding.

Reply:
Right, I'll add something in and fix the typo.  Thanks.